css/style.css

```diff
@@ -0,0 +1,1095 @@
+}
+
+.sm-container {
+  float: left;
+  margin-left: 30px;
+  margin-right: 30px;
+  margin-bottom: 180px;
+}
+
+.small-container {
+  height: 115.85px;
+  width: 115.85px;
+  background-color: #cecece;
+  transform: rotate(-10deg);
+  border-radius: 5%;
+}
+
+.small-link {
+  text-decoration: none;
+  margin-bottom: 5px;
+  color: black;
+  border-bottom: 1px solid black;
+}
+
+.small-link:active {
+  color: black;
+}
+
+h5 {
+  text-decoration: none;
+  font-weight: bold;
+  font-size: 1.8em;
+  font-family: "Cairo", sans-serif;
+}
+
+.code {
+  background-color: #cecece;
+}
+
+.small-name {
+  font-weight: bold;
+  font-family: "nunito", sans-serif;
+}
+
+.com-pic-b {
+  float: left;
+  height: 289.61px;
+  width: 289.61px;
+  margin-left: 30px;
+  margin-right: 30px;
+  transform: rotate(-10deg);
+  background-color: #cecece;
+  border-radius: 5%;
+}
+
+.com-title {
+  font-family: "Libre Baskerville", serif;
+  font-weight: lighter;
+  margin-bottom: 30px;
+}
+
+.community-small {
+  display: grid;
+  width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr;
+  grid-template-areas:
+    "c1 c2"
+    "c3 c4"
+    "c5 c6";
+}
+
+.community-small p {
+  font-size: large;
+  font-family: "Nunito", sans-serif;
+}
+
+footer h4 {
+  padding-top: 70px;
+  padding-bottom: 40px;
+  color: #fff;
+  transition: color 0.2s ease-out;
+}
+
+.community-small > h4 {
+  font-size: xx-large;
+}
+
+.community-small span {
+  margin: 20px 5px 20px 5px;
+}
+
+.small p {
+  height: auto;
+}
+
+.featured-section {
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  background-color: #b5b6b7;
+}
+
+.main-sect {
+  height: 300px;
+  padding-left: 50px;
+}
+
+.main-sect p,
+.guide-pa,
+.guide p {
+  font-family: "nunito", sans-serif;
+}
+
+.book-image {
+  height: 300px;
+}
+
+.book-image div {
+  position: relative;
+  top: -40px;
+  float: right;
+  height: 300px;
+  width: 220px;
+  margin-right: 50px;
+  transform: rotate(-8deg);
+  background-color: gray;
+}
+
+.small-container:hover,
+.book-image div:hover,
+.com-pic-b:hover,
+.author-photo div:hover,
+.head-article div:hover {
+  transform: rotate(0deg);
+}
+
+.our-guides {
+  display: grid;
+  margin-top: 30px;
+  grid-template-columns: 4% 30% 30% 30%;
+  grid-template-rows: auto;
+  grid-template-areas: "txt g1 g2 g3";
+}
+
+.guide-text {
+  grid-area: "txt";
+}
+
+.guide1 {
+  border: 1px solid black;
+  height: 600px;
+  margin: 60px 20px 60px 20px;
+  border-radius: 10px;
+  grid-area: "g1";
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.guide2 {
+  border: 1px solid black;
+  height: 600px;
+  margin: 60px 20px 60px 20px;
+  border-radius: 10px;
+  grid-area: "g2";
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.guide3 {
+  border: 1px solid black;
+  height: 600px;
+  margin: 60px 20px 60px 20px;
+  border-radius: 10px;
+  grid-area: "g3";
+  padding-left: 20px;
+  padding-right: 20px;
+}
```

Comment:
DRY. All properties(except the `grid-area` property, which are also invalid) in these 3 style definitions are the same. Group them in one style definition.